PropertyChanged.Fody/OnChangedWalker.cs

```diff
@@ -14,12 +14,12 @@ void ProcessOnChangedMethods(List<TypeNode> notifyNodes)
     {
         foreach (var notifyNode in notifyNodes)
         {
-            ProcessOnChangedMethods(notifyNode);
+            notifyNode.OnChangedMethods = GetOnChangedMethods(notifyNode);
             ProcessOnChangedMethods(notifyNode.Nodes);
         }
     }
 
-    void ProcessOnChangedMethods(TypeNode notifyNode)
+    List<OnChangedMethod> GetOnChangedMethods(TypeNode notifyNode)
```

Comment:
It looks like IDictionary<TKey, TValue>.Values is ICollection<TValue> rather than IList<TValue>. 

I've changed TypeNode.OnChangedMethods to be ICollection<OnChangedMethod>.

Reply:
👍